app/models/organization.rb

```diff
@@ -47,7 +47,24 @@ def all
   has_many :vendors, dependent: :destroy
   has_many :storage_locations, dependent: :destroy
   has_many :inventory_items, through: :storage_locations
-  has_many :items, dependent: :destroy
+  has_many :items, dependent: :destroy do
+    def during(date_start, date_end = Time.zone.now.strftime("%Y-%m-%d"))
+      select("COUNT(line_items.id) as amount, name")
+        .joins(:line_items)
+        .where("line_items.created_at BETWEEN ? and ?", date_start, date_end)
+        .group(:name)
+    end
```

Comment:
Ignore my previous (deleted) comment. 

I re-read what I was asking for originally; this is fine!